samcli/commands/check/bottle_necks.py

```diff
@@ -0,0 +1,73 @@
+"""
+Bottle neck questions are asked here. Data is saved in graph, but not calcualted here.
+"""
+
+import click
+
+
+class BottleNecks:
+    def __init__(self, graph):
+        self.graph = graph
+
+    def ask_entry_point_question(self):
+        entry_points = self.graph.get_entry_points()
+
+        # All entry points must be calcualted before info can be displayed
+        while entry_points != []:
```

Comment:
Use `while not entry_points:` instead of `while entry_points != []:` is more pythonic.

Reply:
Shouldn't this be `while entry_points:`, since I want it to run while there are elements in the list?

Reply:
Yep, should be `while entry_points:`.